simGanESS/source/geometries/GaP.hh

```diff
@@ -33,23 +33,46 @@ class GaP1 : public GeometryBase
 
         // Materials
         G4Material* gas_;
+        G4Material* mesh_mat_;
         G4Material* cath_mat_;
         G4Material* gate_mat_;
         G4Material* anode_mat_;
 
+
         // Vessel parameters
+        G4double vessel_out_rad_   ;
+        G4double vessel_out_length_;
         G4double vessel_rad_   ;
-        G4double vessel_thickn_;
         G4double vessel_length_;
+        //G4double vessel_thickn_;
 
         // Meshes
-        G4double cath_thickn_;     
-        G4double gate_thickn_;     
-        G4double anode_thickn_;     
-        G4double cath_transparency_; 
-        G4double gate_transparency_; 
-        G4double anode_transparency_; 
+        G4double mesh_rad_   ;
+        G4double mesh_length_;
```

Comment:
This name is not really meaningful so change it to something that indicates better what this is.